src/util/Twitch.hpp

```diff
@@ -4,6 +4,8 @@
 
 namespace chatterino {
 
+QString const TWITCH_USERNAME_PATTERN("^[a-zA-Z0-9][a-zA-Z0-9_]{1,24}");
```

Comment:
```suggestion
QString const TWITCH_USERNAME_PATTERN("^[a-zA-Z0-9][a-zA-Z0-9_]{0,24}$");
```

To ensure length cannot exceed 25 chars & allows for one-character usernames (like [i](https://www.twitch.tv/i/about))